### PR TITLE
Tip search and article view

### PR DESCRIPTION
**Pull request by wvictor-rr**

### CODE REVIEW

File: app/src/main/java/com/nike/dooit/views/main/fragments/TipsFragment.java [hunk 1]
```diff
@@ -4,23 +4,33 @@
...
 import javax.inject.Inject;
 
 import butterknife.BindString;
 import butterknife.BindView;
 import butterknife.ButterKnife;
 
-public class TipsFragment extends Fragment {
+public class TipsFragment extends Fragment implements OnTipsAvailableListener {
 
     private static final String TAG = "TipArchive";
```
Comment (wscheffer) [inline app/src/main/java/com/nike/dooit/views/main/fragments/TipsFragment.java]: TAGs are usually set to TipsFragment.class.getName()

File: app/src/main/java/com/nike/dooit/views/main/fragments/TipsFragment.java [hunk 2]
```diff
@@ -68,7 +83,19 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
...
 
-        TipsTabAdapter adapter = new TipsTabAdapter(getChildFragmentManager(), getContext());
+        searchAdapter = new TipsAutoCompleteAdapter(getContext(), R.layout.list_tips_item);
+        searchView.setAdapter(searchAdapter);
+        searchView.setOnItemClickListener(new AdapterView.OnItemClickListener() {
+            @Override
+            public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
+                Tip tip = searchAdapter.getItem(position);
+                Toast.makeText(getContext(),
+                        String.format("Opening '%s'", tip.getTitle()), Toast.LENGTH_SHORT).show();
```
Comment (wscheffer) [inline app/src/main/java/com/nike/dooit/views/main/fragments/TipsFragment.java]: Should be declared as a String Resource for Local

File: app/src/main/java/com/nike/dooit/views/main/fragments/tip/TipViewHolder.java [hunk 3]
```diff
@@ -65,7 +67,12 @@ public Context getContext() {
 
     @OnClick(R.id.card_tip_title)
     public void startArticle(View view) {
-        Toast.makeText(view.getContext(), titleView.getText().toString() + " clicked", Toast.LENGTH_SHORT).show();
+        Toast.makeText(view.getContext(),
```
Comment (wscheffer) [inline app/src/main/java/com/nike/dooit/views/main/fragments/tip/TipViewHolder.java:81]: Should be declared as a String Resource for Local

**Review by wscheffer — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/nike/dooit/views/main/fragments/tip/TipViewHolder.java (hunk 3 above)
Comment (wvictor-rr) [reply app/src/main/java/com/nike/dooit/views/main/fragments/tip/TipViewHolder.java:81]: Done

**Pull request merged**